odl/solvers/advanced/proximal_operators.py

```diff
@@ -375,6 +377,113 @@ def _call(self, x, out):
     return _ProximalConvConjL1
 
 
+# TODO: move notes to ODL doc
+def proximal_convexconjugate_kl(space, lam=1, g=None):
```

Comment:
I don't think so.
On the other hand, shortening `convexconjugate` to `convexconj` wouldn't hurt.